spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/TorchSpec.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.torch
+
+import org.scalatest.FlatSpec
+
+class TorchSpec extends FlatSpec {
+  def torchCheck(): Unit = {
+    if (!TH.hasTorch()) {
```

Comment:
Output of ```
if (!TH.hasTorch()) {
      cancel("Torch is not installed")
}
```
is something like
```
load torch's Inception+bn float version
- should generate correct output !!! CANCELED !!!
  Torch is not installed (InceptionSpec.scala:607)
```

output of `assume(TH.hasTorch(), "Torch is not installed")` is something like
```
AlexNet float
- should generate correct output !!! CANCELED !!!
  com.intel.analytics.bigdl.torch.TH.hasTorch() was false Torch is not installed! (AlexNetSpec.scala:36)
```
The former is better I think.